src/client/components/app/Transactions/StakeTx.tsx

```diff
@@ -0,0 +1,61 @@
+import { FC } from 'react';
+import { t } from 'i18next';
+
+import { AmountInput } from '@components/app';
```

Comment:
I tend to be cautious when importing from the same folder that will export the component since circular dependencies are hard to debug. Use relative paths in that cases to avoid later on:   
```suggestion
import { AmountInput } from '../AmountInput';
```
note: you will also need to move it down our import groups since we have a lint rule

Reply:
Good catch, refactored with https://github.com/yearn/yearn-finance-v3/pull/741/commits/a6c4a934a78a59a0f92e863bbedde545413432df